src/terminal_theme.rs

```diff
@@ -118,18 +215,16 @@ fn cosmic_light() -> Colors {
     colors[NamedColor::Foreground] = colors[NamedColor::Black];
     colors[NamedColor::Background] = colors[NamedColor::BrightWhite];
     colors[NamedColor::Cursor] = colors[NamedColor::Black];
-    /*TODO
-    colors[NamedColor::DimBlack] = colors[NamedColor::];
-    colors[NamedColor::DimRed] = colors[NamedColor::];
-    colors[NamedColor::DimGreen] = colors[NamedColor::];
-    colors[NamedColor::DimYellow] = colors[NamedColor::];
-    colors[NamedColor::DimBlue] = colors[NamedColor::];
-    colors[NamedColor::DimMagenta] = colors[NamedColor::];
-    colors[NamedColor::DimCyan] = colors[NamedColor::];
-    colors[NamedColor::DimWhite] = colors[NamedColor::];
-    */
-    colors[NamedColor::BrightForeground] = colors[NamedColor::BrightWhite];
-    //TODO colors[NamedColor::DimForeground] = colors[NamedColor::];
+    colors[NamedColor::BrightForeground] = colors[NamedColor::BrightBlack];
```

Comment:
If this is the light theme it should be BrightWhite

Reply:
Yeah, sorry. I forgot to mention that I changed this.

I changed it because I thought it was wrongly set to `BrightWhite`. Is that not the case?

Reply:
I guess you are right, it should not be BrightWhite for the BrightForeground.